syntax = "proto3";

package redisimport; 

service RedisImport {
  
  rpc Import (User) returns (ImportReply) {}
  // rpc ImportBulk (stream User) returns (ImportReply) {}
}

message User {
  double id = 1;
  string first_name = 2;
  string last_name = 3;
  string email = 4;
  string gender = 5;
  string ip_address = 6;
  string user_name = 7;
  string agent = 8;
  string country = 9;
}

// The response message containing the greetings
message ImportReply {
  string message = 1;
  string user = 2;
}